From: SeongJae Park <sj@kernel.org>
Date: Fri, 28 Nov 2025 09:04:39 -0800
Subject: [PATCH] Docs/admin-guide/mm/damon/usage: add a blueprint of
 faults-based monitoring

The current faults-based monitoring for write-only and cpu-specific
monitoring is using a very temporal interface.  Apparently that is not
good enough.  Add another interface blueprint on the document.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/admin-guide/mm/damon/usage.rst | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/admin-guide/mm/damon/usage.rst b/Documentation/admin-guide/mm/damon/usage.rst
index 7da4c002cb39..d5e618eb4e2d 100644
--- a/Documentation/admin-guide/mm/damon/usage.rst
+++ b/Documentation/admin-guide/mm/damon/usage.rst
@@ -65,6 +65,9 @@ comma (",").
     │ │ │ │ │ :ref:`monitoring_attrs <sysfs_monitoring_attrs>`/
     │ │ │ │ │ │ intervals/sample_us,aggr_us,update_us
     │ │ │ │ │ │ │ intervals_goal/access_bp,aggrs,min_sample_us,max_sample_us
+    │ │ │ │ │ │ access_cheeck/primitives/page_table_scan,page_faults
+    │ │ │ │ │ │ access_cheeck/filters/nr_filters
+    │ │ │ │ │ │ │ 0/matching,allow,cpu_id,pid,memcg_path,write,
     │ │ │ │ │ │ nr_regions/min,max
     │ │ │ │ │ :ref:`targets <sysfs_targets>`/nr_targets
     │ │ │ │ │ │ :ref:`0 <sysfs_target>`/pid_target,obsolete_target
-- 
2.47.3

